crates/common/src/path.rs

```diff
@@ -1,11 +1,12 @@
 use std::{fs::create_dir_all, io};
 
 use normpath::BasePathBuf;
+#[allow(clippy::module_name_repetitions)]
 pub use normpath::{error::*, BasePath as Path, BasePathBuf as PathBuf};
 use thiserror::Error;
 
 #[derive(Error, Debug)]
-pub enum PathsError {
+pub enum Error {
```

Comment:
I think the previous, more specific name was better, especially because it's public and can be imported anywhere, is there some specific reason for the rename?

Reply:
I was following the pedantic Clippy lints. The rule `module_name_repetitions` I thought I'd try it and see. Some crates use `Error` and some use `CrateNameError` type names. There does not seem to be clear consensus? Although the `std` crate uses `Error`, e.g. `std::io::Error`, `std::fmt::Error`. Serde and detour also use this convention.

I'm a novice, I don't have much of an opinion either way.

Reply:
That sounds OK
What I'd probably do as a consensus is import it using `use common::path::Error as PathError` in other modules so it's more difficult to confuse it

Reply:
I updated references to the Error objects to either use an alias as per your example, or use the crate path.